File: beginner/javascript/2313.js
var input = require("fs").readFileSync("stdin", "utf8");
var lines = input.split(" ");

var a, b, c;

a = parseInt(lines[0]);
b = parseInt(lines[1]);
c = parseInt(lines[2]);

if(b + c > a && a + c > b && a + b > c){
    if (a == b && b == c) {
        console.log("Valido-Equilatero");
    } else if (a == b || b == c || a == c) {
        console.log("Valido-Isoceles");
    } else {
        console.log("Valido-Escaleno");
    }

    if (a * a == b * b + c * c || b * b == a * a + c * c || c * c == a * a + b * b) {
        console.log("Retangulo: S");
    } else {
        console.log("Retangulo: N");
    }
}
else{
    console.log("Invalido");
}

File: beginner/python/2313.py
a, b, c = list(map(int, input().split()))

if(b + c > a and a + c > b and a + b > c):
    if (a == b and b == c):
        print("Valido-Equilatero")
    elif (a == b or b == c or a == c):
        print("Valido-Isoceles")
    else :
        print("Valido-Escaleno")
   

    if (a * a == b * b + c * c or b * b == a * a + c * c or c * c == a * a + b * b):
        print("Retangulo: S")
    else:
        print("Retangulo: N")
   
else:
    print("Invalido")      